docs/content/latest/drivers-orms/java/postgres-jdbc.md

```diff
@@ -0,0 +1,319 @@
+| :---------- | :---------- | :------ |
+| ssl  | Enable SSL JDBC Connection | false
+| ssl-mode |  SSL mode used for JDBC Connection | require
+| sslrootcert | Server CA Certificate | root.crt
+
+#### SSL Modes
+
+| SSL Mode | Client Driver Behavior | YugabyteDB Support |
+| :------- | :--------------------- | ------------------ |
+| disable  | SSL Disabled | supported
+| allow    | SSL enabled only if server requires SSL connection | Not supported
+| prefer | SSL enabled only if server requires SSL connection | Not supported
+| require | SSL enabled for data encryption and Server identity is not verified | supported
+| verify-ca | SSL enabled for data encryption and Server CA is verified | Supported
+| verifiy-full | SSL enabled for data encryption. Both CA and hostname of the certificate are verified | Supported
+
+#### JDBC Client Identity Authentication
+
+YugabyteDB cluster can be configured to authenticate the identity of the JDBC clients connecting to the cluster. In such cases, server certificate (`yugabytedb.crt`) and server key (`yugabytedb.key`) are required along with root certificate (`ca.crt`).
+
+Steps for Configuring the JDBC Client for Server authentication,
+
+1. Download the certificate (`yugabytedb.crt`, `yugabytedb.key`, and `ca.crt`) files (see [Copy configuration files to the nodes](../../../../secure/tls-encryption/server-certificates/#copy-configuration-files-to-the-nodes)).
+
+2. If you do not have access to the system `cacerts` Java truststore you can create your own truststore.
+
+    ```sh
+    $ keytool -keystore ybtruststore -alias ybtruststore -import -file ca.crt
+    ```
+
+3. Verify the `yugabytedb.crt` client certificate with `ybtruststore`.
+
+    ```sh
+    $ openssl verify -CAfile ca.crt -purpose sslclient tlstest.crt
+    ```
+
+4. Convert the client certificate to DER format.
+
+    ```sh
+    $ openssl x509 –in yugabytedb.crt -out yugabytedb.crt.der -outform der
+    ```
+
+5. Convert the client key to pk8 format.
+
+    ```sh
+    $ openssl pkcs8 -topk8 -inform PEM -in yugabytedb.key -outform DER -nocrypt -out yugabytedb.key.pk8
+    ```
+
+Create an `ssl` resource directory in your java application and copy over all the certificates. Update the connection string used by `DriverManager.getConnection` to include the ssl ceritifcates.
+
+Example JDBC URL for connecting to Secure YugabyteDB cluster can be seen below.
+
+```java
+Connection conn = DriverManager.getConnection(jdbc:postgresql://localhost:5433/yugabyte?ssl=true&sslmode=verify-full&sslcert=src/main/resources/ssl/yugabytedb.crt.der&sslkey=src/main/resources/ssl/yugabytedb.key.pk8", "yugabyte", "yugabyte");
+```
+
+| JDBC Params | Description | Default |
+| :---------- | :---------- | :------ |
+| ssl  | Enable SSL JDBC Connection | false
+| ssl-mode |  SSL mode used for JDBC Connection | require
+| sslcert | Client SSL Certificate | yugabytedb.crt
+| sslkey | Client SSL Certificate Key | yugabytedb.key
+
+## Transaction and Isolation Levels
```

Comment:
Belongs in Reference section? This page should just be the CRUD example?

Reply:
Yes, I have made code changes to remove this from this page.